### PR TITLE
Updated Meetings.md

### PR DESCRIPTION
**Pull request by tonit**
Added Recording Link for call on Feb 11th.
Also added a hint on the probably general monthly cadence of those meetings.

### CODE REVIEW

File: MEETINGS.md [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
 
 ## Upcoming Meetings
-Thursday, February 11th at 18:00 GMT
-[Video Conference Link](https://weaveworks.zoom.us/j/93181235360)<br>
-Meeting ID: 93181235360
-<br><br><br>
+Proposed next meeting: We need to pick regular monthly cadence to be on CNCF calendar -2nd thursday 6pm GMT
 
 ## Past Meetings
+[Video Recording](https://weaveworks.zoom.us/rec/share/OPy5MSMzgIkPX1-PAdhwrnGGa82Lr5BuVpQ1iSK-sbHPph68ML1JNVQXRKlP09bw.bjqDnPKEA2W9D2HG)<br>
```
Comment (chrisk700) [inline MEETINGS.md:8]: Should the date/time be added as per the other recordings?

i.e. Thursday, February 11th at 18:00 GMT

File: MEETINGS.md [hunk 2]
```diff
@@ -2,12 +2,14 @@
 Please find meetings of the GitOps Working Group available below. The next meeting will be listed at the top of the list with links to recordings of past meetings available below (when possible). Times listed are GMT.
 
 ## Upcoming Meetings
-Thursday, February 11th at 18:00 GMT
-[Video Conference Link](https://weaveworks.zoom.us/j/93181235360)<br>
-Meeting ID: 93181235360
-<br><br><br>
+Proposed next meeting: We need to pick regular monthly cadence to be on CNCF calendar -2nd thursday 6pm GMT
```
Comment (todaywasawesome) [inline MEETINGS.md:5]: We do have a monthly cadence which is the 2nd Thursday 6pm GMT as you point out.

**Review by todaywasawesome — CHANGES_REQUESTED** [review]

**Comment by tonit** [comment]
Upcomings updated to reflect what we know: "every 2nd Thursday 6pm GMT". Is there a link to a cncf calendar entry & fixed meeting link?

**Comment by cdavisafc** [comment]
I'm drawing these changes and those from PR #60 into a new consolidated one.

**Comment by tonit** [comment]
Thanks for heads-up, @cdavisafc 👍🏼

**Pull request closed without merging**